### ISSUE TITLE
Social Bookmark Plugin does not work in latest version 3932

### ISSUE THREAD

**Issue by pendolino** [quicksilver/Quicksilver#1080]
says current architecture unsupported. i used this quite a bit to access my Pinboard.in bookmarks.

**Comment by nickv2002** [comment]
I assume that Social Bookmarks for QS is dead since it doesn't seem to have been updated for a while.  I tired to install it but didn't see anywhere in the UI to configure it to my Pinboard.in account (or Del.icio.us).

Is anyone here still using it successfully?

If it's dead, why still have it in the plugin list that is shown from the QS prefs?

**Comment by pjrobertson** [comment]
You’re right it’s still appearing in the plugin list. If you say it doesn’t work then it should definitely be removed. We can do that.
Do any parts of the plugin work? Have you read the manual on the plugin? https://qsapp.com/docs/Quicksilver.pdf <https://qsapp.com/docs/Quicksilver.pdf> (page 86)

FYI: version 3932 isn’t the latest version of Quicksilver. You update to the latest version at http://qsapp.com/download.php

**Comment by nickv2002** [comment]
Thanks @pjrobertso 

I was able to get a little farther in the configuration thanks to the manual (I didn't find that Social Bookmarks Catalog item before because it's buried under a sub-menu). However, even after configuring the plugin as described in the manual (including changing my password to be less than 8 characters with no symbols), it doesn't create any catalog items:

![social bookmarks](https://cloud.githubusercontent.com/assets/704392/15368657/d24e30da-1ce3-11e6-9f07-6bdbdc0c8d17.png)

I don't have a delicious or magnolia account to test those options but the Pinboard one seems to not be working.

Also, FWIW, I'm using QS build 4019, which is current as of this post.

Thanks for your help.

**Comment by rectalogic** [comment]
This should be working now (tested QS build 401A with Social Bookmarks 2.1.0)

The issue for me at least was api.pinboard.in was not using PFS (perfect forward secrecy) with SSL  and apple ATS requires that now. I had tested using `nscurl --ats-diagnostics` and it failed, see the log here https://gist.github.com/rectalogic/c2b0f9fe746d7790f4b2c196c0d9de88

I contacted the pinboard maintainer and he updated the SSL configuration and now Social Bookmarks plugin is able to populate the catalog with all my bookmarks.

I don't think pinboard tags are working properly with the plugin though.
